src/Features/Core/Portable/UnusedReferences/UpdateAction.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.UnusedReferences
+{
+    internal enum UpdateAction
+    {
+        /// <summary>
+        /// No action needs to be performed.
+        /// </summary>
+        None,
+
+        /// <summary>
+        /// Indicates the reference needs to be updated.
+        /// </summary>
+        Update,
```

Comment:
Couldn't we have TreatAsUsed and TreatAsUnused be two different update actions?

Reply:
Sure, splitting into two